eng/Versions.props

```diff
@@ -273,6 +273,7 @@
     <NewtonsoftJsonBsonVersion>1.0.2</NewtonsoftJsonBsonVersion>
     <NewtonsoftJsonVersion>12.0.2</NewtonsoftJsonVersion>
     <NSwagApiDescriptionClientVersion>13.0.4</NSwagApiDescriptionClientVersion>
+    <PlaywrightSharpVersion>0.180.0</PlaywrightSharpVersion>
```

Comment:
How frequently should we be updating this?

We had a build ops task to update the Selenium dependencies on a monthly cadence. Wondering if something similar is needed here or if we can be hands free with it.

Reply:
Something like monthly I think, but I'm not worried right now since we are going to be tinkering with it.